app/components/UI/Ramp/Deposit/Views/BasicInfo/BasicInfo.tsx

```diff
@@ -240,17 +269,32 @@ const BasicInfo = (): JSX.Element => {
       setLoading(false);
     }
   }, [
-    previousFormData,
     validateFormData,
     formData,
     postKycForm,
     submitSsnDetails,
     navigation,
     quote,
+    previousFormData,
     selectedRegion?.isoCode,
     trackEvent,
   ]);
 
+  const handleLogout = useCallback(async () => {
+    try {
+      await logoutFromProvider(false); // false = no server invalidation needed
+
+      // Navigate back to email entry screen
+      navigation.navigate(...createEnterEmailNavDetails({}));
```

Comment:
No need to pass empty params.
```suggestion
      navigation.navigate(...createEnterEmailNavDetails());
```